### PR TITLE
Add Application Guard for Office as a requirement

### PR DESCRIPTION
**Pull request by itguysocal**
In the list of requirements, it was missing Application Guard as a pre-requisite. Since Safe Documents uses hardware based virtualization, it requires Application Guard to be deployed as a prerequisite.

### CODE REVIEW

File: microsoft-365/security/office-365-security/safe-docs.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ ms.prod: m365-security
 Safe Documents is a premium feature that uses the cloud backend of [Microsoft Defender for Endpoint](/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-advanced-threat-protection) to scan opened Office documents in [Protected View](https://support.microsoft.com/office/d6f09ac7-e6b9-4495-8e43-2bbcdbcb6653) or [Application Guard for Office](https://support.microsoft.com/topic/9e0fb9c2-ffad-43bf-8ba3-78f785fdba46).
 
 Users don't need Defender for Endpoint installed on their local devices to get Safe Documents protection. Users get Safe Documents protection if all of the following requirements are met:
-
+- Deploy [Application Guard for Office](https://support.microsoft.com/topic/9e0fb9c2-ffad-43bf-8ba3-78f785fdba46)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/safe-docs.md]:
```suggestion
- [Application Guard for Office](https://support.microsoft.com/topic/9e0fb9c2-ffad-43bf-8ba3-78f785fdba46) is deployed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @itguysocal, kindly commit the suggested change. Many thanks.

**Comment by itguysocal** [comment]
Thanks Johan, I have committed the suggested change

**Comment by chrisda** [comment]
@itguysocal, I asked the experts and they say that Safe Docs does NOT require Application guard. In fact, they said that it's really the other way around: Safe Docs is a prerequisite for Application Guard. But, since this is a Safe Docs topic, we don't really need to say that in the topic.

So, I'm closing this PR.

Thanks for the feedback.

**Pull request closed without merging**